### PR TITLE
Fix typos introduced typos in `popup_menu_theme_test.dart`

### PR DESCRIPTION
**Pull request by ValentinVignal**
There are no linked issues but it comes from the review in https://github.com/flutter/flutter/pull/110268.

Some typos were introduced in https://github.com/flutter/flutter/pull/110268, this PR fixes them.

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
~- [ ] I listed at least one issue that this PR fixes in the description above.~
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Review by goderbauer — APPROVED** [review]
LGTM

Thank you!

File: packages/flutter/test/material/popup_menu_theme_test.dart [hunk 1]
```diff
@@ -83,8 +83,8 @@ void main() {
         child: Column(
           children: <Widget>[
             Padding(
-              // The padding makes sure the menu as enough space to around to
-              // get properly aligned when displayed (`_kMenuScreenPadding`).
+              // The padding makes sure the menu has enough space to around it
```
Comment (goderbauer) [inline packages/flutter/test/material/popup_menu_theme_test.dart:86]: remove the "to" between "space" and "around"?

File: packages/flutter/test/material/popup_menu_theme_test.dart [hunk 2]
```diff
@@ -225,8 +225,8 @@ void main() {
...
```
Comment (goderbauer) [inline packages/flutter/test/material/popup_menu_theme_test.dart:228]: same

**Review by goderbauer — CHANGES_REQUESTED** [review]

File: packages/flutter/test/material/popup_menu_theme_test.dart (hunk 1 above)
Comment (ValentinVignal) [reply packages/flutter/test/material/popup_menu_theme_test.dart]: Thanks, I've removed it in https://github.com/flutter/flutter/pull/111060/commits/4d310b3c6f6d18f1f58b211ea0940fd907a281b0

File: packages/flutter/test/material/popup_menu_theme_test.dart (hunk 2 above)
Comment (ValentinVignal) [reply packages/flutter/test/material/popup_menu_theme_test.dart]: Same, I removed it in https://github.com/flutter/flutter/pull/111060/commits/4d310b3c6f6d18f1f58b211ea0940fd907a281b0

**Review by goderbauer — APPROVED** [review]
LGTM

**Review by chunhtai — APPROVED** [review]
LGTM

**Review by HansMuller — APPROVED** [review]
LGTM

**Pull request merged**